pkgs/games/harmonist/default.nix

```diff
@@ -0,0 +1,34 @@
+{stdenv, fetchurl, buildGoPackage}:
+
+buildGoPackage rec {
+
+  pname = "harmonist-${version}";
```

Comment:
pname shouldn't contain version, that's the point with pname, it's used to build name by combining pname with version

Reply:
Thanks, I didn't find it in the documentation. Fixed. I hope it's all right now. Do I understand correctly that a name is optional if a pname is defined?

Reply:
It was a convention in the python ecosystem within nix that has spread to stdenv.mkDerivation and can be used in most places. Not mandatory but nice :)

Reply:
Oh, I got it. Thanks :)